### PR TITLE
docs: hoist ADR status

### PR DESCRIPTION
**Pull request by rootulp**
Addresses https://github.com/celestiaorg/celestia-app/pull/1161#issuecomment-1437293103

## Questions
1. Do we want to make the same change across other celestiaorg repos? 
2. Should we consider an org wide ADR template?
    1. https://github.com/celestiaorg/celestia-app/issues/1415

### CODE REVIEW

File: docs/architecture/adr-001-abci++-adoption.md
Comment (musalbas) [inline docs/architecture/adr-001-abci++-adoption.md:5]: What does that mean? Is it accepted or only proposed?

File: docs/architecture/adr-007-universal-share-prefix.md
Comment (musalbas) [inline docs/architecture/adr-007-universal-share-prefix.md:5]: Implemented is not a real status according to ADR template. Do we want to add implemented as a status type, and if so, should we update all the implemented Accepted statuses to Implemented?

File: docs/architecture/adr-011-optimistic-blob-size-independent-inclusion-proofs-and-pfb-fraud-proofs.md
Comment (musalbas) [inline docs/architecture/adr-011-optimistic-blob-size-independent-inclusion-proofs-and-pfb-fraud-proofs.md:7]: Bit messy imo to have half-accepted ADRs..

File: docs/architecture/adr-template.md
Comment (musalbas) [inline docs/architecture/adr-template.md:7]: Implemented is not here - should it be?

**Comment by musalbas** [comment]
Left some general comments about inconsistent usage of status message but it doesn't have to block this PR which just moves the status field up.

File: docs/architecture/adr-007-universal-share-prefix.md
Comment (rootulp) [reply docs/architecture/adr-007-universal-share-prefix.md:5]: > Do we want to add implemented as a status type

Sure.

> should we update all the implemented Accepted statuses to Implemented?

Will do in this PR.

File: docs/architecture/adr-template.md
Comment (rootulp) [reply docs/architecture/adr-template.md:7]: It should be because the text above it:

> Once the ADR has been implemented mark the ADR as "implemented"

Comment (rootulp) [reply docs/architecture/adr-template.md]: Resolved by 7d564a1ff4c571744d63825869b246cc543ce01f

**Comment by rootulp** [comment]
I looked through Github repos in celestiaorg but couldn't find an adequate place for an ADR template. @MSevey or @liamsi do you have a suggestion for where such an ADR template should live?

File: docs/architecture/adr-001-abci++-adoption.md
Comment (rootulp) [reply docs/architecture/adr-001-abci++-adoption.md]: Resolved by 34ed8282e7571fe3c105799bdb7840785af39f4e

File: docs/architecture/adr-005-qgb-reduce-state-usage.md
Comment (rootulp) [inline docs/architecture/adr-005-qgb-reduce-state-usage.md:5]: @SweeXordious is it safe to update this to Accepted or Implemented? I'm not confident because the `## Decision` section has incomplete checkboxes

File: docs/architecture/adr-template.md
Comment (rootulp) [inline docs/architecture/adr-template.md:7]: this list doesn't contain "Deprecated" or "Superseded" either.

File: docs/architecture/adr-005-qgb-reduce-state-usage.md
Comment (SweeXordious) [reply docs/architecture/adr-005-qgb-reduce-state-usage.md:5]: `This one is huum, I implemented an MVP for it in [here](https://github.com/sweexordious/celestia-app/tree/qgb_working_draft_of_new_ingestion_2.0) but we never really had a deployed version. Instead, we stopped working on it and switched to [this](https://github.com/celestiaorg/orchestrator-relayer/pull/66). 

Can we put in the status something like `replaced by deciding to use a P2P network` or similar?

File: docs/architecture/adr-template.md
Comment (rootulp) [reply docs/architecture/adr-template.md]: Resolved by https://github.com/celestiaorg/celestia-app/pull/1407/commits/be4369590deba49f6fab881aaa7988b22edfd2fe

File: docs/architecture/adr-007-universal-share-prefix.md
Comment (rootulp) [reply docs/architecture/adr-007-universal-share-prefix.md:5]: Resolved by https://github.com/celestiaorg/celestia-app/pull/1407/commits/ce7c3bebcd9fd762e9d635d437a15e07f6ed8710

File: docs/architecture/adr-005-qgb-reduce-state-usage.md
Comment (rootulp) [reply docs/architecture/adr-005-qgb-reduce-state-usage.md:5]: Yea, we have a Deprecated option. Is there a doc I can link to for the P2P network?
```suggestion
Deprecated
```

**Comment by MSevey** [comment]
> I looked through Github repos in celestiaorg but couldn't find an adequate place for an ADR template. @MSevey or @liamsi do you have a suggestion for where such an ADR template should live?

Since the ADRs are files, I don't know if any GitHub features for the templates, like issue templates for example. 

The simplest thing would be to just have an adr_template.md file in each repo. It's mostly a one time cost to make the template in all the repos that need it.

File: docs/architecture/adr-005-qgb-reduce-state-usage.md
Comment (SweeXordious) [reply docs/architecture/adr-005-qgb-reduce-state-usage.md:5]: It's this PR: https://github.com/celestiaorg/orchestrator-relayer/pull/66 that contains the ADr, which is still not merged

Comment (rootulp) [reply docs/architecture/adr-005-qgb-reduce-state-usage.md]: Resolved by f34bc88914e437c58aafb1f2039d2da11c308071

**Comment by rootulp** [comment]
We already have an ADR template per repo so changes to the ADR template (like the one in this PR) would need to be re-applied across all repos. I was proposing a celestiaorg wide repo that contains one ADR template. It may also be useful for things like linter rules where we establish an org wide standard.

File: docs/architecture/adr-005-qgb-reduce-state-usage.md
Comment (SweeXordious) [inline docs/architecture/adr-005-qgb-reduce-state-usage.md:47]: [Question][Non-blocking]

What's the difference between `[...](<link>)` and `[...](link)`?

Comment (rootulp) [reply docs/architecture/adr-005-qgb-reduce-state-usage.md:47]: there isn't a difference. My editor auto surrounded the link with `<` and `>` likely to conform to https://github.com/updownpress/markdown-lint/blob/master/rules/034-no-bare-urls.md but in this case, it's not a bare URL because of the preceding `[here]`

**Comment by nashqueue** [comment]
We should maybe include authors / Co-authors in ADRs  so they can be pinged in future references

**Review by evan-forbes — APPROVED** [review]

**Comment by evan-forbes** [comment]
org wide template would be cool

**Review by SweeXordious — APPROVED** [review]
LGTM, would  defer to other reviewers for final approval

**Pull request merged**
